src/Domain/DoctrineDonationRepository.php

```diff
@@ -741,6 +741,33 @@ public function findAllCompleteForCustomer(StripeCustomerId $stripeCustomerId):
         return $result;
     }
 
+    /**
+     * We only set Payment Intent on the day of the payment due to stripe limitations, see
+     */
+    public function findDonationsToSetPaymentIntent(\DateTimeImmutable $atDateTime, int $limit): array
```

Comment:
This seems to have no test coverage – would be a good fit for an integration test?